StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/New Payment Method Screen/AddPaymentMethodViewController.swift

```diff
@@ -59,6 +59,10 @@ class AddPaymentMethodViewController: UIViewController {
     private var paymentMethodFormElement: PaymentMethodElement {
         paymentMethodFormViewController.form
     }
+    
+    private var incentive: PaymentMethodIncentive? {
+        elementsSession.linkSettings?.linkConsumerIncentive.flatMap { PaymentMethodIncentive(from: $0) }
```

Comment:
Minor: Since we do this getter more than once, it might be nice to add `incentive` as a computed property on `linkSettings` or maybe even `elementsSession`